usr/share/rear/layout/save/GNU/Linux/220_lvm_layout.sh

```diff
@@ -251,9 +251,9 @@ local lvs_exit_code
     # that the recreated system did not boot (boot screen showed GRUB but there it hung with constant 100% CPU usage)
     # so automatically shrinking only the biggest LVs avoids that a relatively small 'swap' LV gets shrinked.
     # With 'sort -n -t ':' -k 4' the above 'lvs' output lines become
-    #   :swap:system:1262485504:linear::0:1:0:1262485504
-    #   :home:system:6148849664:linear::0:1:0:6148849664
-    #   :root:system:14050918400:linear::0:1:0:14050918400
+    #   :swap:system:1262485504:linear::0:1:0:1262485504:
+    #   :home:system:6148849664:linear::0:1:0:6148849664:
+    #   :root:system:14050918400:linear::0:1:0:14050918400:
```

Comment:
So the `lv_metadata_size` is actually an empty string? What kind of values do we typically find there? Maybe better use a real-world example?

Reply:
I think @pcahyna only wanted to adapt my simple example
(which contains already some other empty values)
to the new syntax that has now for simple LVM setups
an additional empy value at the end.

Reply:
the example is not useful for the case of thin pools, as it does not show a thin pool configuration. `lv_metadata_size` is a non-empty string only for a thin pool.

By the way, I suspect the example is wrong, the numbers do not match the `# With the above example size=`... comment.

Reply:
Likely the values in the example are no longer right.
I did that step by step while I overhauled the whole stuff.